go-update/src/main/java/org/reactome/release/goupdate/DuplicateReporter.java

```diff
@@ -0,0 +1,101 @@
+package org.reactome.release.goupdate;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+import org.gk.model.GKInstance;
+import org.gk.model.ReactomeJavaConstants;
+import org.gk.persistence.MySQLAdaptor;
+import org.gk.schema.GKSchemaAttribute;
+
+/**
+ * This class reports on duplicate GO Accessions.
+ * @author sshorser
+ *
+ */
+public class DuplicateReporter
+{
+	private static String duplicateAccessionQuery = "select accession, count(db_id) as instance_count\n" + 
+			"from (\n" + 
```

Comment:
This 'from' doesn't connect to anything I can see?

Reply:
It is selecting from the subquery (which is a union of three different tables), which is between parentheses.

Reply:
Ah of course, BEDMAS

Reply:
:+1:  :)